### ISSUE TITLE
warning: Member try (variable) of namespace lib_validation is not documented.

### ISSUE THREAD

**Issue by splint33** [doxygen/doxygen#9942]
Hello everyone, I am currently using Doxygen 1.9.7 (Dev build with the fix for the issue 9839).
I am using it to generate a documentation on multiple python custom packages.
On one of them, I think there is an issue.

There is an extract of the __init__.py file of my package:

```
# -*-coding:utf-8 -*

##	@file		__init__.py 
##	@date		XXXX/XX/XX
##	@author		XXX, XXX
##	@brief		XXX
##	
##	@details	XXX


##	@package	XXX
##	@brief		XXX


# Python library modules importation
#	-- None --
# Third party modules importation 
import urllib3
#	Project modules importation 
XXX


try:
	.......
except TestConfigError:
	.......
```

When generating the documentation, Doxygen raise a warning on the try statement.
It looks like it is interpreting it as a variable:

> __init__.py:40: warning: Member try (variable) of namespace XXX is not documented.

I don't think there is a need in documenting a try statement in Python, but I could be wrong.

**Comment by albert-github** [comment]
Indeed there is no need to document the `try:`, doxygen sees it as a variable:
![image](https://user-images.githubusercontent.com/5223533/226983873-49fd2b38-248b-4fd5-bb19-cebcc847c119.png)

Please always mention the full doxygen version, especially when using the master / development version (i.e. the result of `doxygen -v`).

Problem is reproducible with a default Doxyfile in version 1.9.7 (2ac024c1505acdd2d7cfaba66ac818a592b71fc2

**Comment by albert-github** [comment]
In the doxygen code I see that lines starting with an identified followed by a color (`:`) are to be considered as a variable like in e.g.
```
    nsmap = {
        prefix: uri
        for node in rootNode.iter()
        for (prefix, uri) in node.nsmap.items()
        if prefix is not None
    }
```
looks like that an exception has to be made for `try:` and `else:` and ...

**Comment by splint33** [comment]
Just tested with your patch (1.9.7 (b4e95d34ecf85c5e77a897650f56da86199cd7ba*)), it seems to resolve the issue !

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.7.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
